docpipeline/generate.py

```diff
@@ -137,9 +165,11 @@ def setup_docfx(tmp_path, blob):
         metadata_path = decompress_path.joinpath("docs.metadata")
         text_format.Merge(metadata_path.read_text(), metadata)
 
-    metadata.xrefs[:] = [
-        get_xref(xref, blob.bucket, tmp_path) for xref in metadata.xrefs
-    ]
+    # TODO: Consider xrefs for local deployment
```

Comment:
What happens if we try to `get_xref` when running locally? There is no bucket so it fails?

I think it would be better if we logged a warning about any xrefs that couldn't be fetched locally. In the future, we can implement a way for people to provide an `xrefs` directory where they've downloaded the xref files or something.

If we don't log a warning, the xrefs won't be processed and it could lead to confusing output.

Also, there are non-bucket xrefs we should probably still plumb through.

Reply:
any calls to a bucket class member would throw an exception when running locally, but taking a look at `get_xref` it simply would return an empty string for every xref so it shouldn't do much either way. If there are any xrefs stated in `docs.metadata*` it will retain the original list of xrefs supplied in the metadata. 

I think supplying a separate list of xrefs should be better if there are more xrefs to be considered.

Reply:
Will you add a warning log message saying xrefs are being ignored?

Reply:
Done!